src/components/TransactionsFees/index.tsx

```diff
@@ -3,22 +3,30 @@ import Paragraph from 'src/components/layout/Paragraph'
 import { getNativeCurrency } from 'src/config'
 import { TransactionFailText } from 'src/components/TransactionFailText'
 import { Text } from '@gnosis.pm/safe-react-components'
+import useCanTxExecute from 'src/logic/hooks/useCanTxExecute'
+import { providerSelector } from 'src/logic/wallets/store/selectors'
+import { useSelector } from 'react-redux'
+import { currentSafe } from 'src/logic/safe/store/selectors'
+import { checkIfOffChainSignatureIsPossible } from 'src/logic/safe/safeTxSigner'
 
 type TransactionFailTextProps = {
   txEstimationExecutionStatus: EstimationStatus
   gasCostFormatted?: string
   isExecution: boolean
   isCreation: boolean
-  isOffChainSignature: boolean
 }
 
 export const TransactionFees = ({
   gasCostFormatted,
   isExecution,
   isCreation,
-  isOffChainSignature,
   txEstimationExecutionStatus,
 }: TransactionFailTextProps): React.ReactElement | null => {
+  const { currentVersion: safeVersion = '' } = useSelector(currentSafe)
```

Comment:
You should not default the Safe version otherwise it won't calculate the possibility of off chain signing correctly. You should get it from the Safe instance.